Formula/envd.rb

```diff
@@ -0,0 +1,31 @@
+class Envd < Formula
+  desc "Reproducible development environment for AI/ML"
+  homepage "https://envd.tensorchord.ai"
+  url "https://github.com/tensorchord/envd/archive/v0.2.5.tar.gz"
```

Comment:
Should we update it when there is a new release?

Reply:
No, it will be handled by a batch script automatically(see https://github.com/Homebrew/homebrew-core/pull/118117). Though I will add the completion script after https://github.com/tensorchord/envd/pull/1303 is released.

Reply:
Someone will still need to run `brew bump-formula-pr` to create the update though.